src/GraphQL/Execution/DocumentExecuter.cs

```diff
@@ -89,7 +89,7 @@ public async Task<ExecutionResult> ExecuteAsync(ExecutionOptions options)
 
                 if (operation == null)
                 {
-                    throw new ExecutionError("Unable to determine operation from query.");
+                    throw new InvalidOperationException($"Query does not contain operation '{options.OperationName}'.");
```

Comment:
I want to draw your attention to a problem in terminology. This issue was discussed in the GraphQL spec repository. Formally, the request text cannot be identified with the query. We must not forget that there are mutations and subscriptions. This error message can be confusing in case of mutation or subscriptions. One could say `GraphQL request does not contain...` or something similar. In any case, this remark refers rather to the general approach of composing message texts, and not to a specific piece of code. I am ok with the text that is now. If you want you can create a new issue about such wording.

Reply:
👍 This can be reviewed at the same time as the quotes vs double quotes

Reply:
See #1775